CHANGELOG.md

```diff
@@ -1,5 +1,107 @@
 # Lotus changelog
 
+# UNRELEASED
```

Comment:
is this line still suppose to be here?

Reply:
Yes, it should always be there for any UNRELEASED changes